syntax = "proto3";
package object_micro_app;




import "micro_app_sdk/model/micro_app/object_micro_app.proto";

/*
AggregateMicroApp请求
*/
message AggregateMicroAppRequest {
message Query {
    /*
    按objectId列表查询
    */
    repeated string objectId = 1 ;
}
        
    
    /*
    查询条件
    */
    Query query = 1 ;
}

/*
AggregateMicroApp返回
*/
message AggregateMicroAppResponse {
message List {
    /*
    模型ID
    */
    string objectId = 1 ;
    /*
    关联小产品列表
    */
    repeated micro_app.ObjectMicroApp microAppList = 2 ;
}
        
    
    /*
    数据列表
    */
    repeated List list = 1 ;
}

/*
AggregateMicroAppApi返回
*/
message AggregateMicroAppResponseWrapper {
    /*
    返回码
    */
    int32 code = 1 ;
    /*
    返回码解释
    */
    string codeExplain = 2 ;
    /*
    错误详情
    */
    string error = 3 ;
    /*
    返回数据
    */
    object_micro_app.AggregateMicroAppResponse data = 4 ;
}

